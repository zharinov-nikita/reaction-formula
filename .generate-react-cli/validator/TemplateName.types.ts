export interface ValidatorTemplateNameProps {}
export interface ValidatorTemplateName {}
