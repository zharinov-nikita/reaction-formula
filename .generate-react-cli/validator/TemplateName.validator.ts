import { ValidatorTemplateName, ValidatorTemplateNameProps } from './template-name.types'

export function validatorTemplateName(props: ValidatorTemplateNameProps): ValidatorTemplateName {
  return {}
}
