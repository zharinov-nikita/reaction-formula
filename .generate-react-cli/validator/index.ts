export * from './template-name.validator'
export * from './template-name.types'