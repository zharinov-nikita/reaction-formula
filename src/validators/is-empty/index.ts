export * from './is-empty.validator'
export * from './is-empty.types'
