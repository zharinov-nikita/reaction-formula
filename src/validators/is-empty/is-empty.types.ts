export type ValidatorIsEmptyProps = string
export type ValidatorIsEmpty = boolean
