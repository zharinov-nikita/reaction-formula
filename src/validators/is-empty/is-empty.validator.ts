import { ValidatorIsEmpty, ValidatorIsEmptyProps } from './is-empty.types'

export function validatorIsEmpty(value: ValidatorIsEmptyProps): ValidatorIsEmpty {
  return value.length === 0
}
