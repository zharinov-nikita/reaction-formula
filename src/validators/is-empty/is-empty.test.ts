import { validatorIsEmpty } from './is-empty.validator'

describe('validatorIsEmpty', () => {
  test('expected true', () => {
    expect(validatorIsEmpty('')).toBe(true)
  })

  test('expected false', () => {
    expect(validatorIsEmpty('not empty')).toBe(false)
  })

  test('snapshot', () => {
    expect(validatorIsEmpty('')).toMatchSnapshot()
    expect(validatorIsEmpty('not empty')).toMatchSnapshot()
  })
})
