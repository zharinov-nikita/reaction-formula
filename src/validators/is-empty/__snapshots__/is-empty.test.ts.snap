// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`validatorIsEmpty snapshot 1`] = `true`;

exports[`validatorIsEmpty snapshot 2`] = `false`;
